projects/online/online/utils/pe.py

```diff
@@ -91,54 +121,29 @@ def postprocess_samples(
         idx = inference_params.index(param)
         posterior[param] = samples.T[idx].flatten()
 
+    # TODO remove
+    posterior["phi"] = ra
     posterior["ra"] = ra
     posterior["dec"] = dec
     posterior = pd.DataFrame(posterior)
 
-    result = bilby.result.Result(
+    result = AmplfiResult(
         label=f"{event_time}",
         posterior=posterior,
         search_parameter_keys=inference_params,
     )
     return result
 
 
-def create_histogram_skymap(
-    ra_samples: np.ndarray,
-    dec_samples: np.ndarray,
-    nside: int = 32,
-) -> tuple[table.Table, plt.Figure]:
-    """Create a skymap from samples of right ascension
-    and declination using a naive histogram estimator.
-    """
-    # mask out non physical samples;
-    mask = (ra_samples > -np.pi) * (ra_samples < np.pi)
-    mask &= (dec_samples > 0) * (dec_samples < np.pi)
-
-    ra_samples = ra_samples[mask]
-    dec_samples = dec_samples[mask]
-    num_samples = len(ra_samples)
-
-    # calculate number of samples in each pixel
-    npix = hp.nside2npix(nside)
-    ipix = hp.ang2pix(nside, dec_samples, ra_samples)
-    ipix = np.sort(ipix)
-    uniq, counts = np.unique(ipix, return_counts=True)
-    uniq_ipix = nest2uniq(nside, np.arange(npix))
-
-    # create empty map and then fill in non-zero pix with counts
-    m = np.zeros(npix)
-    m[np.in1d(range(npix), uniq)] = counts
-
-    post = m / num_samples
-    post /= hp.nside2pixarea(nside)
-    post /= u.sr
-
-    # convert to astropy table
-    t = table.Table(
-        [uniq_ipix, post],
-        names=["UNIQ", "PROBDENSITY"],
-        copy=False,
+# TODO: need more robust way to
+# specify the parameter sampler,
+# either from the config,
+# or by loading in from checkpoint
+def parameter_sampler() -> ParameterSampler:
```

Comment:
Is this function still getting used? I see that you added `amplfi_parameter_sampler` to the config file.

Reply:
yeah it still is. I updated the config